src/neo/Cryptography/MPT/MPTTrie.cs

```diff
@@ -9,21 +10,22 @@ public partial class MPTTrie<TKey, TValue>
         where TValue : class, ISerializable, new()
```

Comment:
Are `TKey` and `TValue` realy used? I saw you use `byte[]` and `ReadOnlySpan<byte>` directly in most case.

Reply:
Only in arguments of `Put` `Get` `Delete`.

Reply:
I use `ToArray()` at start in all case.